### ISSUE TITLE
Document where to raise issues

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-forge.github.io#487]
It's not totally obvious that policy issues should be raised on the webpage repo, or that package requests should show up on staged-recipes, or what the feedstocks repo issue tracker is used for. We should document this somehow so new people understand where to get help and feedback on ideas they have.

**Comment by jakirkham** [comment]
Also would be good to link external issue trackers that are relevant to. For example, `conda` and `conda-build` as well as things like [Anaconda Platform's support issue tracker]( https://github.com/Anaconda-Platform/support/issues ).

**Comment by PrernaSingh587** [comment]
Has this been incorporated? @jakirkham @viniciusdc

**Comment by viniciusdc** [comment]
Hi @PrernaSingh587 no it still not incorporated, feel free to open a PR if you are interested :smile:, this in especifiq I will not assign to anyone as it is now, but feel free to link your related PR's with this.

**Comment by PrernaSingh587** [comment]
@viniciusdc  Sure I will work on it!

**Comment by viniciusdc** [comment]
> @viniciusdc Sure I will work on it!

btw, have a good look inside the main parts of the docs, because (maybe) some parts of this issue are already done, but scattered inside the docs

**Comment by PrernaSingh587** [comment]
@jakirkham 
policy issues should be raised on the webpage repo,

or that package requests should show up on staged-recipes,

or what the feedstocks repo issue tracker is used for.


In this issue, @viniciusdc mentioned that some part of it is already done. I gather , by digging the docs, I saw that the 2nd ( https://conda-forge.org/docs/maintainer/infrastructure.html#repositories) and 3rd(https://conda-forge.org/docs/orga/getting-in-touch.html?highlight=issue%20tracker) part is already done.
I can't seem to find anything related to the first part i.e the policy issues.

**Comment by ssurbhi560** [comment]
@kathatherine @jakirkham, I am thinking of adding the remaining things (issue tracker for ``conda``, ``conda-build`` and [Anaconda Platform's support issue tracker](https://github.com/Anaconda-Platform/support/issues) in the "Getting in touch" section.  Also, I think this could also be a good idea to have a section entirely for this, something like **Raise your issues/concerns** in "Getting in touch". Please share what do you think about this, if this sounds good, I will open a PR for this.

**Comment by kathatherine** [comment]
@ssurbhi560, did you end up reopening this PR anywhere else? This would be good to finish up.

**Comment by jakirkham** [comment]
Sorry for the slow reply.

Yeah maybe this is already covered with "Getting in Touch". Though Idk if this should live in "Organisation Documentation" or "User Documentation". Maybe the latter is friendlier for newer contributors? What do you think?

Policy issues probably make sense on the webpage repo. That's where we have had these historically. Though maybe we can spell that out better.

**Comment by kathatherine** [comment]
I agree that Getting in Touch would probably work better in the User Documentation. Do y'all do redirects to make sure no outside links are broken when whole sections are moved?

**Comment by jakirkham** [comment]
We have [this]( https://github.com/conda-forge/conda-forge.github.io/blob/09bc2d08f0e3ebe583621054a8bda52cf878e75b/rtd/conf.py#L302-L309 ), which maybe we can extend

WDYT @beckermr?

**Comment by beckermr** [comment]
I don't know how the redirects work. Sounds like a good idea.

**Comment by ssurbhi560** [comment]
@kathatherine, I haven't reopened it yet. And now that we are planning to move the "Getting in Touch" section to "User documentation" I would need to understand how redirects work. I will look into this and will open a PR soon.

> Policy issues probably make sense on the webpage repo. That's where we have had these historically. Though maybe we can spell that out better.

Hi @jakirkham! :)
Just to clarify, by webpage repo, do you mean on the website, [here](https://conda-forge.org/)?
